Project1/InventoryViewable.cpp

```diff
@@ -36,4 +36,3 @@ int InventoryViewable::might(Passkey<Stats> key) const {
 int InventoryViewable::atk_spd(Passkey<Stats> key) const {
 	return stats_help(_owner.getStats().getAttribs(), &ItemEquip::atk_spd);
 }
-
```

Comment:
![alt text](http://i0.kym-cdn.com/entries/icons/original/000/022/978/yNlQWRM.jpg "y tho")

Reply:
It was an accidental change when I was looking through files lol